### DISCUSSION TITLE
wireguard-ui

### DISCUSSION THREAD

**Discussion by ElInquilino** [wg-easy/wg-easy#592] [Ideas]
Great work !.
Could you analyze the idea of including a more complete gui for wg-easy like ngoduykhanh wireguard-ui ?
Thanks in advance.

**Comment by pheiduck** [comment]
The idea behind wg-easy is to keep it as simple as possible. Overloading the GUI would contradict that I think. @WeeJeWel 
And the other project fulfills your requirements. You are not forced to use wg-easy. :)
